xblockmufi/templates/view.html

```diff
@@ -0,0 +1,61 @@
+<div class="xblockmufi_block">
+    <h2>{{ display_name }}</h2>
+    <div class='your_answer' style='display:none'>{{ your_answer_label }}</div>
+    <div class="xblockmufi_block_textarea txtEditor">
```

Comment:
Let's make this div (and everything it contains) a separate template and include it in both edit and view templates, so we duplicate less code.

Reply:
You'll probably want to make the file something like `_text_editor.html`...

Reply:
How would I be able to import this new html file inside the two files?

Reply:
Would this {% include %} line go at the start of the file?